README.md

```diff
@@ -1,11 +1,283 @@
+
+## Main SDK Features <a name="features"></a>
+* Playback of one or more individual videos or a single playlist of videos
+* Video playback of VOD (video on demand), 360°, and LIVE streaming video types
+* Supports either .mp4 or .m3u8 (HLS) formats
+* Video ads (VAST support of .mp4 format only)
+* Tap an ad to access ad URL (more information) via an in-app-browser
+* Full video and ads analytics
+* Default video player controls UX (full source code open sourced)
+* HLS support for multiple closed captioning (CC) and SAP audio languages
+* Mute/Unmute support
+* Automatic filtering of geo-restricted videos
+* Complete apps control of the frame where videos play
+* Native iOS Picture-in-picture support
+* Apple AirPlay support
+
+## Why would I use the O2 Mobile SDK? <a name="why"></a>
+The O2 Mobile SDK is used to natively play O2 videos. If you have a native app, you should use the SDK. The main reason as to why you’d want to use the SDK, is because you get all the ads and analytics for free. The ads are important for monetization. Analytics are important for tracking your app’s video and video ads performance and usage. This helps you understand what your users are watching with your app, and how much.
+
+There are several technical advantages to using the native OMSDK over a web player-based solution. We won’t go into these in depth in this document, but here are some of the advantages:
+* Performance
+* Mobile network awareness
+* Frugal memory, thread, and device resource usage
+* Security – webviews / embedded browsers are known weak points
+* Fine-grained control, less limits
+* Some platforms don’t have webviews (e.g., Apple TV)
+* More customization options
+
+## Advertising Info and User Tracking <a name="privacy"></a>
+The O2 Mobile SDK does not track anything that is not related to playing videos or video ads. We use the IDFA (ID for advertisers) value and respect the user's settings for Limit Ad Tracking (iOS enforces this anyway). The device geolocation is determined by our backend video servers based on IP address, for the purposes of determining and filtering out content that is geo-restricted by content owners. The SDK does not explicitly use the built-in Location Services APIs, and thus does not require your users to grant access to device location data.
+
+## Starting Requirements <a name="requirements"></a>
+* **Apple Xcode 8+**
+* **Swift 3.1 or 3.2***
+* **CocoaPods or Carthage**
+* **Mobile device running iOS 8 or later or AppleTV device running tvOS 9 or later**
+* **Account in the O2 Portal, and access to Oath-ingested video content**
+* **Onboarded application bundle ID**
+
+**\* Note**: Swift 4 support is coming early in Q1 of 2018.
```

Comment:
This looks like a plan - better to update the doc when we have Swift 4 support.

Reply:
remove word "early" to be safe.